swarm.go

```diff
@@ -556,9 +582,25 @@ func (s *Swarm) RegisterPssProtocol(topic *pss.Topic, spec *protocols.Spec, targ
 // serialisable info about swarm
 type Info struct {
 	*api.Config
-	*chequebook.Params
 }
 
-func (s *Info) Info() *Info {
-	return s
+func (i *Info) Info() *Info {
+	return i
+}
+
+type SwapInfo struct {
```

Comment:
this should be the API and live within swap packagev

Reply:
and why params are used with pointer semantics

Reply:
Ah the same as I comment here? https://github.com/ethersphere/swarm/pull/1554/files#r308403651